source/standards/threat-modelling.html.md.erb

```diff
@@ -0,0 +1,236 @@
+---
+title: Threat Modelling
+last_reviewed_on: 2021-06-25
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## What's a threat?
+
+A threat is an action that causes denial, alteration or disclosure of your
+assets that you are trying to protect. It could be a user database, a deployment
+pipeline or the integrity of web form submissions.
+
+## What's threat modelling?
+
+Threat modelling is analysing a service to raise potential information security
+and privacy issues.
+
+The best time to perform threat modelling activities is during the design phase;
+however, it can be done anytime and should become a continuous process in your
+service team.
+
+Within the Cabinet Office and GDS, we follow the [Threat Modeling Manifesto]'s
+four questions:
+
+1. What are we working on?
+2. What can go wrong?
+3. What are we going to do about it?
+4. Did we do a good enough job?
+
+We answer most of the above over three sessions, with time between allowing for
+asynchronous input.
+
+### 1. Threat Discovery
+
+Threat discovery is looking to answer the “What are we working on?” and part of
+the “What can go wrong?” question.
+
+We use an interactive exercise to talk through and explain the system or
+architecture. Someone from the service team will need to lead this part. Try to
```

Comment:
Does it matter which role the person has in the service team?

Reply:
I don't think so - I've had design leads, PMs, architects, and devs do this role well.
Maybe saying something like:
```suggestion
architecture. Someone who is comfortable with explaining the system from the service team will need to lead this part. Try to
```

Reply:
Yes, I think that works